### DISCUSSION TITLE
.orgue format

### DISCUSSION THREAD

**Discussion by thbeckpr** [GrandOrgue/grandorgue#1638] [Help]
I would like to modify the ODF of a sample set I have in the .orgue format. 

How can I extract the files into directories so that I can use the .organ format?

**Comment by ClarkeBrunt** [comment]
.orgue is just a zip file. So you should be able to open/extract it with whatever zip program you use. On Windows I can do e.g. Open with... 7-zip (which I've installed). Or (I just tried it), rename the file with .zip on the end, and Windows own file explorer can look into it.

I think it's documented in GrandOrgue help precisely what a .orgue is (you'd probably have to read that if re-making a .orgue from the contents).

**Reply by thbeckpr** [reply]
I'm using Linux, specifically Linux Mint 21.1. The default archive manager for this distribution will not open the .orgue file. Does anyone have any recommendations for a Linux archive manager which will read the .orgue file?

**Reply by jason-barba** [reply]
gzip / gunzip

//j

**Reply by larspalo** [reply]
The default File Roller that is integrated in Nautilus file browser on Ubuntu opens it fine. You could try renaming the .orgue extension to .zip and see if that helps with your archive manager. Read the documentation in the GO help for details about the .orgue format, as not just any zip option will do when creating the archive (even though that's not what you're trying to do right now)...

**Reply by jason-barba** [reply]
My suggestion was directed at us old school shell users ;-)

//j

**Reply by larspalo** [reply]
> My suggestion was directed at us old school shell users ;-)

Still works today! Actually, to create the .orgue (.zip) file with the proper parameters, I still use the terminal...

**Reply by thbeckpr** [reply]
The shell script unzip worked for me.
